requirements.txt

```diff
@@ -26,12 +26,12 @@ django-debug-toolbar==1.6
 django-dotenv==1.4.1
 django-easy-pdf==0.1.1
 django-localflavor==1.4.1
-django-modelcluster==2.0
+django-modelcluster>=3.0,<=4.0
```

Comment:
should be `<` not `<=`?

Reply:
also i don't know if this is intentional or not, but the other cases are updating to a certain minor version branch such as `>= 1.0,<1.1`. however this is updating to a certain major version branch, `>=3.x,<4.x`. any reason for this?